pathd/path_pcep_pcc.c

```diff
@@ -651,6 +667,9 @@ void pcep_pcc_pathd_event_handler(struct ctrl_state *ctrl_state,
 		PCEP_DEBUG("%s Candidate path %s removed", pcc_state->tag,
 			   path->name);
 		path->was_removed = true;
+		/* Removed as response to a PcInitiated 'R'emove*/
+		/* RFC 8291 #5.4 LSP Deletion*/
+		path->do_remove = path->srp_id ? true : false;
```

Comment:
s/RFC8291/8281/

Otherwise, you must check the 'R' flag of the SRP Object, not the SRP-ID itself to decide if the path shoudl be created (R flag unset) or remove (R flag set). This is completely independent from the value of the SRP-ID which serves to identify the sequence of the message exchange between a PCE and a PCC

Reply:
Sure, will change that.